src/components/FeaturedProducts/FeaturedProducts.jsx

```diff
@@ -1,19 +1,18 @@
 import React, { useEffect, useState } from "react";
 import axios from "axios";
 import "./FeaturedProducts.css";
-import {Card} from '../';
+import { Card } from "../";
 export const FeaturedProducts = () => {
+  const [featuredproducts, setFeaturedProducts] = useState([]);
 
-const [featuredproducts, setFeaturedProducts] = useState([]);
-  
   useEffect(() => {
     (async () => {
       try {
         const response = await axios.get("/api/products");
         const featProducts = await response.data.products.slice(0, 3);
         setFeaturedProducts(featProducts);
       } catch (error) {
-        console.log(error);
+        console.error(error);
```

Comment:
I would suggest you load products at once. You can use the products from the product list and use them here.

Reply:
I planned to do it later but forgot. Will do it now. Thanks for reminding.